client/me/privacy/main.jsx

```diff
@@ -56,23 +56,72 @@ const Privacy = createReactClass( {
 			TRACKS_OPT_OUT_USER_SETTINGS_KEY
 		);
 
+		const cookiePolicyLink = (
+			<a href="https://www.automattic.com/cookies" target="_blank" rel="noopener noreferrer" />
```

Comment:
Do folks think an [external link](https://wpcalypso.wordpress.com/devdocs/design/external-link) component would be more appropriate?

Reply:
I think it would be more appropriate: just because it transfers the responsibility of setting  `noopener noreferrer` to it :) I don't think the extra gridicon would fit well though, so I didn't include it.